components/header.tsx

```diff
@@ -45,6 +70,32 @@ export default function Header() {
           <NavLink p={2}>News</NavLink>
         </Link>
       </Flex>
+      <Button
+        className="burger-menu"
```

Comment:
don't use className we have a better way to do that, just stick to js and sx prop